woocommerce/woocommerce-gutenberg-products-block
# Introduce Variation Picker Component

This is the variation picker component (#2398) for the single products block.

This adds new API data (variations and attributes) and finds matches on the client side via a new Variation Attribute Picker Form. This has been added to the Add to Cart Form Atomic Block. It will show attributes and match a variation ID, but is not yet hooked up to the form.

Known follow ups:

- Hooking up to the cart form
- Supporting default attributes

Closes #2398

### How to test

1. Add a single product block for a variable product
2. Confirm the picker is shown in admin and on the frontend, and attributes are selectable on the frontend form.
3. Selecting all attributes should match a variation (this will be shown below the form)

### Screenshots

![Single Product – local wordpress test 2020-06-17 17-34-49](https://user-images.githubusercontent.com/90977/84924895-12465f80-b0c1-11ea-879b-61d349be48b0.png)
